### PR TITLE
docs: improve `/guides/securing-requests/`

### PR DESCRIPTION
**Pull request by Yohe-Am**
Improves `/gudes/securing-requests/`

#### Motivation and context

_N/A_

#### Migration notes

_N/A_

### Checklist

- [ ] The change come with new or modified tests
- [ ] Hard-to-understand functions have explanatory comments
- [ ] End-user documentation is updated to reflect the change

### CODE REVIEW

File: website/docs/guides/external-functions/index.mdx [hunk 1]
```diff
@@ -56,7 +56,7 @@ The following runtimes can be used to run custom functions:
 - [`WasmedgeRuntime`](/docs/reference/runtimes/wasmedge/) through wasm modules.
 - [`PrismaRuntime`](/docs/reference/runtimes/prisma/) throw raw SQL queries.
 
-## The context object
+## Accessing funciton context
```
Comment (Natoandro) [inline website/docs/guides/external-functions/index.mdx:59]: "function"?

File: website/docs/guides/external-functions/index.mdx [hunk 2]
```diff
@@ -43,7 +43,7 @@ The following example uses the `DenoRuntime` to respond to requests and define a
     </TabItem>
 </SDKTabs>
 
-Note for the `fib` root materializer, we're using an ECMA module in an external file.
+Note that for the `fib` root materializer, we're using an ECMA module in an external file.
```
Comment (Natoandro) [inline website/docs/guides/external-functions/index.mdx:46]: But here it's in TypeScript...

File: website/docs/guides/securing-requests/index.mdx [hunk 3]
```diff
@@ -4,38 +4,138 @@ sidebar_position: 50
...
+
+    </TabItem>
+</SDKTabs>
+
+Note, the token is encoded in base64. 
+Decoded, it'd read `andim:password`.
+
+If you were to try to send a request without the header, you'd notice that `get_full_context` still returns a result. 
+An empty object.
+Authentication is only responsible for populating the context object and without a policy to shoot down the reqest, it'll access the materalizers.
```
Comment (Natoandro) [inline website/docs/guides/securing-requests/index.mdx:66]: "request"

**Review by destifo — APPROVED** [review]
Nice! The comments from the RFC are addressed

**Pull request merged**